### PR TITLE
fix lint errors and warnings

### PR DESCRIPTION
**Pull request by iAKM**
- made changes according to the lint errors and warning displayed on running lint test.
- ran unit test cases locally. all test cases have passed.

Note: left with two lint errors which couldn't be resolved.

### CODE REVIEW

**Review by sureshgaussian — CHANGES_REQUESTED** [review]
Please add a message to the PR about what changed, and if unit tests passed in your local machine.

File: src/utility/utility.ts [hunk 1]
```diff
@@ -19,15 +19,14 @@ export class Utility {
         return true;
     }
 
-    public static copy<T extends Object>(target: T, source: any): T {
+    public static copy<T extends object>(target: T, source: any): T {
         Object.keys(target).forEach(key => {
```
Comment (sureshgaussian) [inline src/utility/utility.ts:23]: where is 'Object' from?

**Review by sureshgaussian — COMMENTED** [review]
I am not sure if the code actually works. Did anyone check it? I am running into errors when I type 'npm start' in this branch.

Screenshot 2023-09-13 at 11 57 51 PM



- able to run without any errors.

File: src/utility/utility.ts (hunk 1 above)
Comment (susrisha) [reply src/utility/utility.ts:23]: https://stackoverflow.com/questions/49464634/difference-between-object-and-object-in-typescript

**Review by sureshgaussian — APPROVED** [review]

**Pull request merged**